js/main.js

```diff
@@ -268,3 +284,102 @@ function getAgentData() {
+  $descriptionSectionColHalf.appendChild($descriptionTitle);
+
+  const $weaponDescription = document.createElement('p');
+  let weaponDescription = null;
+  if (weapon.displayName === 'Odin') {
+    weaponDescription = 'A powerful LMG with high damage, high rate of fire, and a large magazine capacity for breaking through defenses.';
+  } else if (weapon.displayName === 'Ares') {
+    weaponDescription = 'A heavy machine gun with a high rate of fire, suitable for suppressing fire and holding positions.';
+  } else if (weapon.displayName === 'Vandal') {
+    weaponDescription = 'A high-damage assault rifle with moderate recoil, capable of eliminating enemies with a single headshot.';
+  } else if (weapon.displayName === 'Bulldog') {
+    weaponDescription = 'A rifle with burst fire mode, offering moderate recoil and decent damage.';
+  } else if (weapon.displayName === 'Phantom') {
+    weaponDescription = 'A fully automatic rifle with low recoil and good accuracy, versatile for close to medium-range encounters.';
+  } else if (weapon.displayName === 'Judge') {
+    weaponDescription = 'A devastating shotgun known for its close-range stopping power and rapid fire rate.';
+  } else if (weapon.displayName === 'Bucky') {
+    weaponDescription = 'A shotgun with a wider spread and moderate damage, effective at close range.';
+  } else if (weapon.displayName === 'Frenzy') {
+    weaponDescription = 'A compact SMG with a high rate of fire but reduced accuracy.';
+  } else if (weapon.displayName === 'Classic') {
+    weaponDescription = 'A versatile starter pistol with burst fire mode for close-range encounters.';
+  } else if (weapon.displayName === 'Ghost') {
+    weaponDescription = 'A silenced pistol with low recoil and good accuracy, ideal for stealthy plays.';
+  } else if (weapon.displayName === 'Sheriff') {
+    weaponDescription = 'A heavy revolver capable of eliminating enemies with a single headshot.';
+  } else if (weapon.displayName === 'Shorty') {
+    weaponDescription = 'A shotgun with limited range but high damage potential at close quarters.';
+  } else if (weapon.displayName === 'Operator') {
+    weaponDescription = 'A powerful sniper rifle that guarantees a one-shot kill regardless of the body part hit.';
+  } else if (weapon.displayName === 'Guardian') {
+    weaponDescription = 'A semi-automatic rifle with high accuracy, suitable for tapping and controlled bursts.';
+  } else if (weapon.displayName === 'Marshal') {
+    weaponDescription = 'A sniper rifle that can eliminate enemies with a single shot to the chest or head.';
+  } else if (weapon.displayName === 'Spectre') {
+    weaponDescription = 'A versatile SMG with good accuracy and manageable recoil for medium-range combat.';
+  } else if (weapon.displayName === 'Stinger') {
+    weaponDescription = 'An SMG with a fast rate of fire, suitable for close to medium-range engagements.';
+  } else if (weapon.displayName === 'Melee') {
```

Comment:
This would be a perfect scenario for a `switch` statement

Reply:
thanks for the advice. ill make sure to edit it during the stretch feature portion of this project